MoneySplitter.Win10/Common/SearchEngine.cs

```diff
@@ -31,6 +37,38 @@ public ObservableCollection<UserModel> Results
             }
         }
 
+        public string MessageForUser
```

Comment:
Doesn't matter for who this message. Rename just to ErrorMessage or something like this

Reply:
rename